stdlib/public/core/SwiftNativeNSArray.swift

```diff
@@ -61,14 +61,16 @@ internal class __SwiftNativeNSArrayWithContiguousStorage
   }
 }
 
+private let NSNotFound: Int = .max
+
 // Implement the APIs required by NSArray 
 extension __SwiftNativeNSArrayWithContiguousStorage: _NSArrayCore {
   @objc internal var count: Int {
     return withUnsafeBufferOfObjects { $0.count }
   }
 
-  @objc(objectAtIndex:)
-  internal func objectAt(_ index: Int) -> AnyObject {
+  @inline(__always)
+  private func _objectAt(_ index: Int) -> AnyObject {
```

Comment:
It may be a good idea to mark this `@nonobjc`, just to make sure it's not exposed there. (I'm not sure if it's relevant for private methods, but I suspect it is.)

Reply:
ah good call

Reply:
Nothing is `@objc` implicitly anymore unless it's in a protocol (or overrides something that's `@objc`, or has one of the other ObjC-ish attrs).